cgi-bin/DW/Controller/Entry.pm

```diff
@@ -740,12 +740,13 @@ sub _form_to_backend {
     $props->{taglist} = "" unless $props->{taglist} && $props->{taglist} =~ /\S/;
 
     if ( LJ::is_enabled( 'adult_content' ) ) {
+        my $restriction_key = $post->{age_restriction} || '';
```

Comment:
Yeah the reason I didn't do that here is all the valid values are enumerated below, they're all either text strings or the empty string.